pex/pip.py

```diff
@@ -483,7 +484,7 @@ def analyze(self, line):
                     via=via,
                 )
             )
-        elif not self._lock_configuration.strict:
+        elif LockStyle.SOURCES == self._lock_configuration.style:
```

Comment:
any particular reason for leading with the enum value rather than the config variable? This reads like defensive Java coding to me rather than solid Python code

Reply:
Its a habit from tests - lead with expected value. Here - if you're used to that style, you can skip reading the RHS of the line - it puts the important bit 1st. Clearly eyes live in beholders though.